internal/services/appservice/windows_function_app_resource.go

```diff
@@ -639,6 +645,10 @@ func (r WindowsFunctionAppResource) Read() sdk.ResourceFunc {
 				DefaultHostname:             utils.NormalizeNilableString(props.DefaultHostName),
 			}
 
+			if hostingEnv := props.HostingEnvironmentProfile; hostingEnv != nil {
+				state.HostingEnvId = utils.NormalizeNilableString(hostingEnv.ID)
```

Comment:
Can we change this to use the `pointer` package as it's already imported? We'll be refactoring all references of this soon, but we shouldn't continue use of  the `utils` package in new changes.